lib/features/presentation/ui/pages/profile_page.dart

```diff
@@ -1,13 +1,123 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_challange/config/theme/icons/app_icons_icons.dart';
+
+const profileLabel = "Profile";
+const profileName = "Ruben Geidt";
+const profileEmail = "ruben.geidt@example.com";
+const profileDetails = "Profile Details";
+const profileSettings = "Settings";
+const profilePush = "Push Notifications";
+const profileSupport = "Support";
+const profileLogout = "Logout";
 
 class ProfileView extends StatelessWidget {
   const ProfileView({super.key});
   @override
   Widget build(BuildContext context) {
-    return const Center(
-      child: Text(
-        'Profile Placeholder',
-        style: TextStyle(fontSize: 24),
+    return Padding(
+      padding: const EdgeInsets.only(top: 8.0, left: 8.0, right: 8.0),
+      child: Column(
+        children: [
+          const Align(
+            alignment: Alignment.topLeft,
+            child: Padding(
+              padding: EdgeInsets.only(left: 11.0),
+              child: Text(
+                profileLabel,
+                style: TextStyle(fontSize: 24, fontWeight: FontWeight.w600),
+              ),
+            ),
+          ),
+          Image.asset('assets/images/profile.png'),
```

Comment:
not accurate, edit icon should be a widget. also i would use CircleAvatar widget (https://api.flutter.dev/flutter/material/CircleAvatar-class.html)